Tests/FluentAssertions.Specs/Equivalency/CollectionEquivalencySpecs.cs

```diff
@@ -715,6 +715,326 @@ public void When_a_deeply_nested_property_of_a_collection_with_an_invalid_value_
             act.Should().NotThrow();
         }
 
+        [Fact]
+        public void When_attribute_in_collection_is_excluded_with_then_excluding_it_should_not_throw()
+        {
+            // Arrange
+            var subject = new
+            {
+                Text = "Text",
+                Level = new
+                {
+                    Collection = new[]
+                    {
+                        new
+                        {
+                            Number = 1,
+                            Text = "Text"
+                        },
+                        new
+                        {
+                            Number = 2,
+                            Text = "Actual"
+                        }
+                    }
+                }
+            };
+
+            var expected = new
+            {
+                Text = "Text",
+                Level = new
+                {
+                    Collection = new[]
+                    {
+                        new
+                        {
+                            Number = 1,
+                            Text = "Text"
+                        },
+                        new
+                        {
+                            Number = 3,
+                            Text = "Actual"
+                        }
+                    }
+                }
+            };
+
+            Action act = () => subject.Should().BeEquivalentTo(expected,
```

Comment:
🙃 (nitpicking) Where's the `// Act/Assert`?

Reply:
✅ Added `// Act/Assert`